packages/rich-text-editor/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -33,6 +33,15 @@ export interface BaseRichTextEditorProps
   rows?: EditorRows
   dataError?: React.ReactElement
   onDataError?: () => void
+  status?: "default" | "error" | "caution"
+  /**
+   * A descriptive message for `error` or `caution` states
+   */
+  validationMessage?: string | React.ReactNode
```

Comment:
You can remove the `string` from the type for both the `description` and the `validationMessage` as the `React.ReactNode` include a string :)